source/stylesheets/_refills-header.scss

```diff
@@ -88,14 +88,33 @@
     @include refills-inset;
     border-radius: 30px;
     color: darken(desaturate($refills-logo-color, 50%), 15%);
-    display: inline;
+    display: inline-block;
     font-family: $refills-narrow-sans;
     font-size: 0.8em;
     font-weight: 400;
+    margin-bottom: 4em;
     padding: 0.5em 1.4em;
 
     @include media($refills-large-screen) {
       font-size: 0.8em;
     }
   }
+
+  .empties-banner {
+    $empties-color: #776faa;
+    background-color: $empties-color;
+    border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border, border-radius, box-shadow, color, display, font-family, padding